src/core/window.ts

```diff
@@ -313,6 +314,18 @@ export abstract class DwmWindow {
     image: ImageStruct,
   ): void;
 
+  /**
+   * Retrieves the window and display/instance handles required for WebGPU
+   * ```ts
+   * const [window, display, instance] = win.rawHandle();
+   * ```
+   */
+  abstract rawHandle(): [
```

Comment:
What if we returned an object instead of array? It would be ideal if Deno.UnsafeWindowSurface accepted the same object (like an options bag instead of three parameters).

Reply:
I think you can return a `Deno.UnsafeWindowSurface` directly from this. And maybe rename to `windowSurface`?

Reply:
That sounds better. Let's do that @load1n9 - `windowSurface` accessor which creates and caches the Deno.UnsafeWindowSurface object internally